<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankOfficial</class>
 <widget class="QMainWindow" name="BankOfficial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankOfficial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenido al banco seleccione una opci√≥n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mostrarcuentas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar cuentas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mostrarclientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Modificardata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar datos del cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Crearcliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Crearcuenta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear cuenta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deposit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Depositar dinero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdrawal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Retirar dinero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Block_Unblock">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Bloquear o desbloquear cuenta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Consulaccount">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar cuenta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Consultarclient">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>180</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar cliente</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
